templates.py

```diff
@@ -3875,6 +3875,48 @@
         ],
     }),
 
+    ('esperanto-verb', {
+        '@attribution': {'users': ['Nikki'], 'title': 'Wikidata:Wikidata Lexeme Forms/Esperanto'},
+        'label': 'esperanta verbo',
+        'language_item_id': 'Q143',
+        'language_code': 'eo',
+        'lexical_category_item_id': 'Q24905',
```

Comment:
```suggestion
        'lexical_category_item_id': 'Q24905',
        'has_sections': True,
```

Reply:
I didn't include that on purpose. 😕

I was hoping to group the fields into sections (infinitive, indicative, other) without columns because there aren't enough fields for multiple columns to produce a table effect, it just looks messy.

If I don't want columns, does that mean I have to remove the sections entirely?

Reply:
I pushed a new commit that allows setting the flag to `False`, to explicitly mark that a template shouldn’t have two-column sections. (Omitting the flag still causes test failures, to prevent that from happening accidentally.)